config/application.rb

```diff
@@ -54,6 +54,7 @@ class Application < Rails::Application
         domain: ENV['MAILGUN_DOMAIN']
     }
 
+    config.active_job.queue_adapter = :sidekiq
```

Comment:
RE sidekiq delay time .... it appears that we have no delay at this time(?).   That's fine as is.   However, perhaps we need an ENV var for the delay (seconds) to give us some flexibility in adjusting this in production (if needed).